docs/samples/accelerators/tensorflow-gpu.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "tensorflow-gpu"
+  annotations:
+    "serving.kubeflow.org/gke-accelerator": "nvidia-tesla-t4"
+spec:
+  default:
+    tensorflow:
+      modelUri: "gs://kfserving-samples/models/tensorflow/flowers"
+      runtimeVersion: "1.11.0-gpu"
+      resources:
+        limits:
+          nvidia.com/gpu: 1
```

Comment:
what`s the rational for using nvidia.com here?

Reply:
s/rational/rationale :)

This is the resource key name standardized by nvidia * k8s:
https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/#v1-8-onwards